runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -230,9 +231,14 @@ public static class CorfuRuntimeParameters extends RuntimeParameters {
         boolean cacheDisabled = false;
 
         /*
-         * The maximum number of entries in the cache.
+         * The maximum number of entries in the AddressSpaceView cache.
          */
-        long maxCacheEntries;
+        long maxCacheEntries = 5000;
+
+        /*
+         * The maximum number of entries in the MVOCache.
+         */
+        long maxMvoCacheEntries = 5000;
```

Comment:
I would tune these defaults down, such that the total  of maxCacheEntries + maxMvoCacheEntries equals the old default. The default behavior should not consume more memory than previous releases.

Reply:
Done. I have divided them evenly at 2500 entries.